### DISCUSSION TITLE
interesting article on energy-efficient risc-v cores

### DISCUSSION THREAD

**Discussion by biosbob** [stnolting/neorv32#643] [General]
found this [here](https://www.mdpi.com/2072-666X/14/7/1371)....

@stnolting -- could a similar technique be used with the `ice*` boards????   if working with `xilinx` boards, it sounds like there are some `vivado` tools to capture switching traces which could then be used in power estimates????

**Comment by stnolting** [comment]
Looks good, thanks for the hint!

Looking just at the core's architecture (which is the "energy efficient" part) I think we could keep up with the power efficiency... :wink:
